### PR TITLE
feat(check-email): add support for manual code input

### PR DESCRIPTION
**Pull request by eliobricenov**
Design: https://www.figma.com/design/GKPELZxk551oVC3EDLgEDA/Current-Design?node-id=13254-72194&m=dev

### CODE REVIEW

File: src/components/CheckEmail/index.tsx [hunk 1]
```diff
@@ -66,21 +80,39 @@ const CheckEmail = ({
...
+        >
+          <TextInput
+            className={styles.input}
+            placeholder={i18n.inputPlaceHolder}
+            {...codeInputProps}
+          />
+        </Collapse>
+      </div>
       <div className={styles.linkContainer}>
         <span className={cn('button-sm', styles.subtitle)}>
```
Comment (BrettCleary) [inline src/components/CheckEmail/index.tsx:117]: minor but I think this typography class is 'caption' on the latest designs

File: src/components/CheckEmail/index.tsx [hunk 2]
```diff
@@ -66,21 +80,39 @@ const CheckEmail = ({
...
-          {i18n.subtitle} <span className="text--semibold">{email}</span>
+        <Modal.Body className={'body-sm'}>
+          {i18n.subtitle} <span className="text--semibold">{email}</span>.{' '}
+          {i18n.clickLoginTitleOr}{' '}
+          <Button
+            type="link"
+            className={styles.reEnter}
+            onClick={onReEnterEmail}
+          >
+            {i18n?.reEnterEmail}.
```
Comment (BrettCleary) [inline src/components/CheckEmail/index.tsx:95]: Same comment as I left on another PR:
Breaking up all the strings will mess up how it gets translated. Using Trans should let you use components while keeping the translation string together https://react.i18next.com/latest/trans-component 

We make the same mistake elsewhere but I'm trying to call it out when I see it

**Review by BrettCleary — APPROVED** [review]
LGTM. just a couple minor comments 🙏

File: src/components/CheckEmail/index.tsx (hunk 2 above)
Comment (eliobricenov) [reply src/components/CheckEmail/index.tsx:95]: nice catch, I'd completely forgotten I did the same here

Comment (eliobricenov) [reply src/components/CheckEmail/index.tsx:93]: We don't have i18n next in the repo though 😅

Comment (eliobricenov) [reply src/components/CheckEmail/index.tsx:93]: what do you think about creating a ticket for this?

Comment (eliobricenov) [reply src/components/CheckEmail/index.tsx:93]: added here https://github.com/HyperPlay-Gaming/hyperplay-ui/issues/384

**Pull request merged**